configs/ACCIF435ELITE/config.h

```diff
@@ -0,0 +1,132 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     AT32F435G
+
+#define BOARD_NAME        ACCIF435ELITE
+#define MANUFACTURER_ID   CUST
+
+#define USE_ACC
+#define USE_GYRO
+
+#define USE_GYRO_CLKIN
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO_SPI_ICM42688P
+
+#define USE_BARO
+#define USE_BARO_DPS310
+#define DEFAULT_BARO_I2C_ADDRESS 119
```

Comment:
default is 0x76 why is this using 0x77 ?

Reply:
Because DPS368 default address when SDO is floating is 0x76, I will add, as sugested, an open bridge so it can be changed if needed if anyone needed.

Reply:
Right, my bad, thanks for the correction :)